example/docker-compose/docker-compose.azure.azurite.yaml

```diff
@@ -0,0 +1,63 @@
+version: "2"
+services:
+
+  tempo:
+    image: grafana/tempo:latest
+    command: ["-config.file=/etc/tempo.yaml"]
+    volumes:
+      - ./etc/tempo-gcs-fake.yaml:/etc/tempo.yaml
```

Comment:
The compose crashes right now with a fake yaml.. this needs to change to `tempo-azure.yaml`